### ISSUE TITLE
Add a Todoist task whenever I tag a zotero entry with a "read" tag

### ISSUE THREAD

**Issue by Fontanapink** [wbthomason/zotodo#10]
It would be great if, whenever I tag a reference entry in Zotero with a "read" or "to-read" tag in Zotero, Zotodo will automatically pick that up and create a Todoist task item out of it.

**Comment by wbthomason** [comment]
Thanks for the feature suggestion! This will mostly entail figuring out how to hook into tag change events in Zotero, then calling the existing task creation function (as well as making a new option for the set of tags to be used as triggers).

**Comment by Fontanapink** [comment]
Hello, any progress in this?

**Comment by wbthomason** [comment]
No, I have not had time to look into it. PRs are welcome if you'd like to take a try yourself!
